components/teams/TeamDetailsForm.js

```diff
@@ -0,0 +1,31 @@
+import { useForm, Controller } from 'react-hook-form'
+import dynamic from 'next/dynamic'
+
+const LocationInput = dynamic(() => import('./LocationInput'), { ssr: false })
```

Comment:
I am seeing a parsing error from vscode here and also in CampaignCard.js

> Parsing error: Unexpected token import

Are you seeing the same @kamicut? It seems the syntax is correct for Next.js and ES2020, but we may need to update either `.eslintrc.json` or open an issue on devseed-standard.

Reply:
@guidorice I'm not using a different eslint, are you getting an error when you run `yarn lint` ?

Reply:
@kamicut ah good point to check: `yarn lint` reports no errors, but in my vscode editor, it's showing errors like:

<img width="408" alt="Screen Shot 2020-02-26 at 2 26 07 PM" src="https://user-images.githubusercontent.com/766758/75389902-9c658080-58a4-11ea-8f82-3538775f52fe.png">

Apparently vscode think's it's running eslint but doing it differently than the devseed-standard script. Are you not seeing the same problem?